src/sentry/models/organization.py

```diff
@@ -442,12 +442,5 @@ def handle_2fa_required(self, request):
             ip_address=ip_address
         )
 
-    def get_url_viewname(self):
```

Comment:
We should to check getsentry for usage of this

Reply:
And maybe even integrations?

Reply:
Good catch. Thanks ❤️